src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryPipeStatus.java

```diff
@@ -266,6 +283,7 @@ public void dumpTo(ObjectNode msg) {
     msg.put(STAGE_NAME, stageName);
     msg.put(PIPE_NAME, pipeName);
 
+    // TODO @rcheng: changed from metricsutil to telemetry?
```

Comment:
Most of these properties referred to the metricsutil property instead of the telemetry property. Does this difference matter? i can change the telemetry property to match the metrics utils property since I'm worried this change might break some dashboards

ex: PROCESSED_OFFSET
`processed-offset` would become -> `processed_offset`

Reply:
yeah lets not change the metrics name, it will break scripts and dashboard.

Reply:
Changed telemetry property to match the metric property